packages/ai-svelte/tests/create-generation.test.ts

```
@@ -7,6 +7,7 @@ import { createSummarize } from '../src/create-summarize.svelte'
 import { createGenerateVideo } from '../src/create-generate-video.svelte'
 import { createMockConnectionAdapter } from './test-utils'
 import { EventType, type StreamChunk } from '@tanstack/ai'
+import type { TTSResult, TranscriptionResult } from '@tanstack/ai'
```

ESLint

[error] '`@tanstack/ai`' imported multiple times.

(import/no-duplicates)